docs/guide/dev/installation/README.md

```diff
@@ -14,23 +14,36 @@ For licensing, see LICENSE.md.
 
 The aim of this article is to get you up and running with CKEditor in two minutes.
 
-## Download
+## Download from Official Site
+
+### Download
 
 Visit the official [CKEditor Download](https://ckeditor.com/ckeditor-4/download/) site. For a production site we recommend you choose the default **Standard Package** and click the **Download CKEditor** button to get the `.zip` installation file. If you want to try out more editor features, you can download the **Full Package** instead.
 
 <a href="https://ckeditor.com/ckeditor-4/download/"><img src="%BASE_PATH%/assets/img/ckeditor_quick_start_download.png" alt="CKEditor Download site" width="914" height="440"/></a>
 
-## Unpacking
+### Unpacking
 
 Unpack (extract) the downloaded `.zip` archive to the `ckeditor` directory in the root of your website.
 
+## Install from the NPM registry
+
+To install [the CKEditor 4 NPM package](https://www.npmjs.com/package/ckeditor):
+
+```bash
+npm install --save ckeditor
+```
+
```

Comment:
You could include link to detailed guide on npm installation here (which is http://localhost:9001/ckeditor4/4.11.4/guide/dev_package_managers.html).

Reply:
And that the install build would be `standard-all`.